critiquebrainz/data/model/user.py

```diff
@@ -95,9 +95,9 @@ def reviews(self):
     def avatar(self):
         """Link to user's avatar image."""
         if self.show_gravatar and self.email:
-            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=mm&r=pg"
+            return "https://gravatar.com/avatar/" + hashlib.md5(self.email.encode("ascii")).hexdigest() + "?d=identicon&r=pg"
         else:
-            return "https://gravatar.com/avatar/placeholder?d=mm"
+            return "https://gravatar.com/avatar/placeholder?d=identicon"
```

Comment:
It seems none of the available types of images in Gravatar support that. 
Just tried with all those here - https://en.gravatar.com/site/implement/images/
We could however generate random hashes for those who don't have an email id to get unique avatars